CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+# Contributing
+
+Everyone is welcome to contribute to GoDaddy's Open Source Software.
+Contributing doesn’t just mean submitting pull requests. To get involved,
+you can report or triage bugs, and participate in discussions on the
+evolution of each project.
+
+No matter how you want to get involved, we ask that you first learn what’s
+expected of anyone who participates in the project by reading the Contribution
+Guidelines and our [Code of Conduct][coc].
```

Comment:
Should "Contribution Guidelines" be a link? Or should this sentence read "these Contribution Guidelines"?

Reply:
Ah yes, I think probably the latter would be more appropriate.